_includes/news.html

```diff
@@ -1,24 +1,24 @@
 <div class="news">
   <h2>news</h2>
-  {% if site.news  %}
+  {%- if site.news  -%} 
```

Comment:
wouldn't adding `-` on both sides place `<h2>news</h2>` and `<div class="table-responsive">` on the same line? (same question about other places where `{%-` and `-%}` where added)

Reply:
Yes, just add the one on the right, not both.

Result:
```html
          <!-- News -->          
          <div class="news">
            <h2>news</h2>
            <div class="table-responsive">
              <table class="table table-sm table-borderless"> 
                <tr>
                  <th scope="row">Jan 15, 2016</th>
                  <td>
                    A simple inline announcement with Markdown emoji! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20">
 
                  </td>
                </tr> 
                <tr>
                  <th scope="row">Nov 7, 2015</th>
                  <td>
                    <a class="news-title" href="/news/announcement_2/">A long announcement with details</a> 
                  </td>
                </tr> 
                <tr>
                  <th scope="row">Oct 22, 2015</th>
                  <td>
                    A simple inline announcement.
 
                  </td>
                </tr> 
              </table>
            </div> 
          </div>
```

I have made more changes so that the HTML output is correct, with 2 spaces (I leave information below). PR: https://github.com/alshedivat/al-folio/pull/481/commits/3808d418d5890ebfc3050a71754d69e04429c103
In Liquid when including html or loading content does not respect indents. Each file must be indented separately.

[Google HTML Formatting](https://developers.google.com/style/html-formatting)
[W3 HTML Style](https://www.w3schools.com/html/html5_syntax.asp)